pages/guide.md

```diff
@@ -20,6 +20,23 @@ You are new to Starcraft II or Protoss? This is the perfect place to start! Keep
 
 This guide assumes you have a basic understanding of Starcraft II, and understand how each unit functions. If you do not, we recommend you start by playing the campaign, an excellent and very fun starter! From there, you can just jump in on ladder and have fun playing against others. After 20 games or so you, should be playing against people of your skill level, where you should win 50% of your games.
 
-From there, you may want to start improving. It is generally agreed upon that the best way to improve is to learn how to macro. What is macro? You can find a very detailed explanation on Terrancraft [here](https://terrancraft.com/2017/05/18/es-mechanics-basics-and-macro/). Basically, it is the actions taken to build an army as big and as quickly as possible, because in Starcraft, a bigger army almost always wins versus a smaller army. How do you macro? The first step is to constantly be building probes, never letting your nexuses go idle until you have a desired count (60-80 probes). Build pylons as to never get supply blocked. With what is left, you build buildings and units.
+## Fundamentals
+
+### Macro
+
+ What is macro? You can find a very detailed explanation on Terrancraft [here](https://terrancraft.com/2017/05/18/es-mechanics-basics-and-macro/). Basically, it is the actions taken to build an army as big and as quickly as possible, because in Starcraft, a bigger army almost always wins versus a smaller army. How do you macro? The first step is to constantly be building probes, never letting your nexuses go idle until you have a desired count (60-80 probes). Build pylons as to never get supply blocked. With what is left, you build buildings and units. Macro is the foundation of Starcraft, and is generally agreed upon as the best way to improve overall.
```

Comment:
```suggestion
 What is macro? You can find a very detailed explanation on Terrancraft [here](https://terrancraft.com/2017/05/18/es-mechanics-basics-and-macro/). Basically, it is the **actions taken to build an army as big and as quickly as possible**, because in Starcraft, a bigger army almost always wins versus a smaller army. How do you macro? The first step is to **constantly be building probes**, never letting your nexuses go idle until you have a desired count (60-80 probes). **Build pylons as to never get supply blocked**. With what is left, you **build buildings and units**. Macro is the foundation of Starcraft, and is generally agreed upon as the best way to improve overall.
```
Added a bit of formatting to have the important parts stand out more!